thoth/adviser/wraps/thoth_search_package.py

```diff
@@ -53,12 +56,18 @@ def should_include(cls, builder_context: "PipelineBuilderContext") -> Generator[
 
     def run(self, state: State) -> None:
         """Add a link to Thoth Search UI for each resolved package."""
+        runtime_environment = self.context.project.runtime_environment
         for package_version_tuple in state.resolved_dependencies.values():
             state.justification.append(
                 {
                     "type": "INFO",
                     "link": self._search_package_url.format(
-                        package_name=package_version_tuple[0], package_version=package_version_tuple[1]
+                        package_name=quote(package_version_tuple[0]),
+                        package_version=quote(package_version_tuple[1]),
+                        index_url=quote(package_version_tuple[2]),
```

Comment:
Search UI encodes the index_url. Use `urllib.parse.quote(package_version_tuple[2], safe="")` for a properly formatted url

Reply:
Oh, nice! Didn't know this was possible. Adjusted. PTAL 👍🏻

Reply:
LGTM